Some improvement on quickstart docs (#4844)
Scille/parsec-cloud: docs/development/quickstart.md

## Hacking the Web client (front)

> In addition to the [shared requirements](#shared-requirements), for working with the web clients you need to:

Reviewer: ```suggestion
In addition to the [shared requirements](#shared-requirements), for working with the web client you need to:
```

Author: For me we have multiple clients `web`, `electron`, `android` & `ios` :thinking:

Reviewer: So "working with the ~web~ clients"? I only suggested that to be coherent with the title (which should probably also be changed to "Hacking the ~web~ client").

IMHO even if the clients share the same core it would seem weird to me to refer to the "android web client", I will prefer "clients" as in "web client", "android client", ...

Author: ```suggestion
In addition to the [shared requirements](#shared-requirements), for working with the clients you need to:
```

Author: Currently the section `Hacking the web client` provide steps for the web & electron, maybe I should split that ?

Reviewer: idk, I guess it's also fine as it is... we can split that later when we add android/ios
So as you wish :)

Author: I will prepare the terrain for it then :)

Author: I've split the section

     - On Linux: you need to have installed `libfuse2` -->

1. Install [`Node 18.12.0`](https://nodejs.org/en/download/releases).

>    We recommand that you use [`nvm`](https://github.com/nvm-sh/nvm) to manage multiple node versions:

Reviewer: ```suggestion
   We recommend using [`nvm`](https://github.com/nvm-sh/nvm) to manage multiple node versions:
```

   > We install it using cargo because it's the simpler way to specify which version of `wasm-pack` to use.

   <!-- markdownlint-disable-next-line no-inline-html -->
3. Initialize a poetry env <a id="init-web-server-env" />

   ```shell
   python ./make.py python-dev-install
   ```

>    > The python backend is build from the folder `server`.

Reviewer: ```suggestion
   > The Python backend is built from the `server` directory.
```

4. Setup the web binding

   ```shell
   python ./make.py web-dev-install
   ```

5. Setup the electron binding

   ```shell
   python ./make.py electron-dev-install
   ```

> 6. Move to the client dir, For the later command we will consider that the current directory is `client`

Reviewer: ```suggestion
6. Move to the `client` directory
```

   ```shell
   cd client
   ```

> 7. Install client dependencies

Reviewer: ```suggestion
7. Install client dependencies from the `client` directory
```
